build.sbt

```diff
@@ -1,19 +1,8 @@
-import com.gu.riffraff.artifact.RiffRaffArtifact
-import com.gu.riffraff.artifact.RiffRaffArtifact.autoImport._
-import play.sbt.PlayImport.PlayKeys._
-
-
 name := """amiable"""
 
 version := "1.0-SNAPSHOT"
 
-lazy val root = (project in file("."))
```

Comment:
Oh, I guess you're just using whole the file as a single project? I prefer to explicitly write the project every time because that approach works in every sbt project, rather than using different approaches for single and multi project builds.